### ISSUE TITLE
heic pictures cannot be sent

### ISSUE THREAD

**Issue by tknupf** [signalapp/Signal-Android#13676]
- [ X] I have searched open and closed issues for duplicates
- [ X] I am submitting a bug report for existing functionality that does not work as intended
- [ X] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ X] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
I cannot send heic pictures created by my stock camera. A bigger picture in jpg works.

### Steps to reproduce
[heic.zip](https://github.com/user-attachments/files/16778588/heic.zip)
Try to send the 2 files in the zip. The jpg works.

**Actual result:** The heic image fails to be send. The jpg works.
**Expected result:** The heic image can be send.

### Screenshots
![Screenshot_20240828_100721_Signal](https://github.com/user-attachments/assets/f1314016-7bb4-4b5c-97b9-7bed83642f72)

### Device info
**Device:** Samsung A52s 5G
**Android version:** 14 Update July (up to date)
**Signal version:** 7.14.2 (up to date)

### Link to debug log
https://debuglogs.org/android/7.14.2/8fdbff1d369495a1d82396e6634f12f295089ca2951e7b7919886cc9a43046d7

https://debuglogs.org/android/7.14.2/7a8c3bdd5303c562f4ce568ae256c302f7fa64c1bca1c59253e525ca5950c9ff

**Comment by olumolu** [comment]
Including avif format

**Comment by tknupf** [comment]
Works now

**Comment by James-h143** [comment]
What did you do to fix? this still appears to be an issue for me

**Comment by tknupf** [comment]
I stumbeled on my ticket today and checked it again. And it just works. Maybe a new version of signal may the os security update...
Feel free to reopen the ticket.

**Comment by christiansblackburn** [comment]
Yeah this is still broken.  My wife and I both have signal and high-end android phones and can't send ~50 MB heic images.

**Comment by trougnouf** [comment]
I am still experiencing this issue.

**Comment by tknupf** [comment]
[bugHeic.zip](https://github.com/user-attachments/files/20205002/bugHeic.zip)
I found another problematic file.

**Comment by tknupf** [comment]
- [ ] 

- [ ] 

- [ ] 

- [ ] vcv

**Comment by trougnouf** [comment]
Even the original file you posted doesn't work for me. I tried uninstalling Signal (from f-droid) and reinstalling from Google Play, that made no difference.
